### ISSUE TITLE
MAS 1.4.3:  Elements must have sufficient color contrast (a[rel="noopener\ noreferrer"][target="_blank"])

### ISSUE THREAD

**Issue by amitkumariiit** [markdown-it/markdown-it#504]
Markdown table have the following accessibility issue. 

![table](https://user-images.githubusercontent.com/9694413/47084712-ff1b2680-d231-11e8-898c-ce54843c4329.png)

Issue Details
Elements must have sufficient color contrast
[color-contrast
](https://dequeuniversity.com/rules/axe/3.1/color-contrast)

Element Path
table:nth-child(111) > tbody > tr:nth-child(6) > td > a[rel="noopener\ noreferrer"][target="_blank"]

Snippet
```
<a href="https://microsoft.sharepoint.com/%3Ab%3A/t/DD_VSCS/EQtiCwq1NRpAjj0_IS5coOYBvnOKL2b5-gAu-3VSLU89OQ?e=RIzVb3" class="" rel="noopener noreferrer" target="_blank">Introduction to platform as a service</a>
```

How to fix
Fix any of the following:
- Element has insufficient color contrast of 4.26 (foreground color: #0078d4, background color: #f8f8f8, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

To reproduce
Use [K4W ](https://microsoft.sharepoint.com/teams/msenable/Pages/KerosForWeb.aspx) to investigate the issue details

Environment
Chrome version 69.0.3497.100

This accessibility issue was found using **Keros for Web**

**Comment by puzrin** [comment]
Close as not related to this project
